lib/pluginmanager/util.rb

```diff
@@ -1,12 +1,35 @@
 require "rubygems/package"
 
 module LogStash::PluginManager
+
+  class Settings
```

Comment:
I kinda like the idea of having this settings in a way similar of what Bundler.settings does. I find it super useful for later invocations or usages of the same parameters, and for actually keeping the knowledge where it belongs. 

In the current implementation the only place where the PluginManager life, is the PluginManager module, there is no other object who might be holding this plugin manager settings. Do you have a proposal where we can keep this? 

I actually think that either naming this module ```PluginManager``` or the file ```utils.rb``` is wrong. Do we aim to have a plugin manager concept? there would be a nice place to hold this settings idea, and many other concepts useful for this codebase.

But this is just my point of view.

The ```Settings``` idea is removed as you requested.